paasta_tools/cli/cmds/spark_run.py

```diff
@@ -65,6 +65,9 @@
 # Extra room for memory overhead and for any other running inside container
 DOCKER_RESOURCE_ADJUSTMENT_FACTOR = 2
 
+# Mass enable DRA configs
+EXECUTOR_RANGES = {(0, 64), (256, 2048)}
```

Comment:
Let's start small? Easy to monitor. 
This should be easy to modify later.

Reply:
Should I just go with `{(0, 64)}`. Or even smaller?

Reply:
Sounds good.